frontend/src/SettingsPage/SettingsPage.jsx

```diff
@@ -106,6 +137,7 @@ function SettingsPage(props) {
                         placeholder="Enter first name"
                         value={firstName}
                         onChange={(event) => setFirstName(event.target.value)}
+                        onKeyPress={(event) => !/^[A-Za-z0-9<XXX> -]+$/.test(event.key) && event.preventDefault(true)}
```

Comment:
Please remove the validation, it should support special characters like é, ñ etc